staubli_tx2-90_support/urdf/tx2-90_macro.xacro

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find staubli_resources)/urdf/common_materials.xacro"/>
+
+  <xacro:macro name="staubli_tx2-90" params="prefix">
+    <!-- links: main serial chain -->
+    <link name="${prefix}base_link">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/visual/base_link.stl"/>
+        </geometry>
+        <xacro:material_staubli_ral_melon_yellow />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/collision/base_link.stl"/>
+        </geometry>
+      </collision>
+      <inertial>
+          <mass value="10.733459"/>
+          <origin xyz="0.014288 0.000001 0.122264" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.096590" ixy="-0.000005" ixz="-0.003803"
+                                  iyy="0.087591"  iyz="-0.000001"
+                                                  izz="0.081632"/>
+      </inertial>
```

Comment:
How did you determine these values?

Reply:
It's written in "readme" part of the package.xml, those values are copied from TX90 package. In the TX90 package those were calculated in SolidWorks assuming homogeneous distribution of mass etc. (by somebody).

I would also be fine with dropping those values to be honest as we don't use them and I have no way to verify them.

Reply:
It would probably be good to add a comment about it here.

We could also remove the elements completely, as they are approximations in any case, and for a different robot.